OrbitGl/TimeGraph.cpp

```diff
@@ -319,6 +319,9 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* fu
     } else {
       scheduler_track_->OnTimer(timer_info);
       cores_seen_.insert(timer_info.processor());
+      uint32_t num_cores = GetNumCores();
```

Comment:
I don't think that you should set the name/number of cores everytime you process a scheduling timer. This should be probably done in the GetOrCreateSchedulerTrack

Reply:
The little issue here is that we don't know the NumCores before processing all timers, do we?

cores_seen_ is simply a set of cores (in my case 8) and we are inserting each processor several times, I changed it, but not sure if you will like it